addons/web/static/src/views/fields/ace/ace_field.js

```diff
@@ -57,11 +56,11 @@ export class AceField extends Component {
     }
 
     commitChanges() {
-        if (!this.props.readonly) {
-            const value = this.state.value;
-            if (this.props.record.data[this.props.name] !== value) {
-                return this.props.record.update({ [this.props.name]: value });
+        if (!this.props.readonly && this.isDirty) {
+            if (formatText(this.props.record.data[this.props.name]) !== this.lastValue) {
```

Comment:
Is it possible to rename `state.value` to `state.initialValue` and `lastValue` to `editedValue`?
In this case, you could replace by `this.state.initialValue !== this.editedValue`.

I think it would be clearer to understand that the update of the edited value is handled by the component, and that an initial value is read from the record